scripts/manage-nodes.sh

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env bash
+
+# Exit script as soon as a command fails.
+set -o errexit
+
+# check for coverage which will specify new port for ethereum client
+if [ "$SOLIDITY_COVERAGE" = true ]; then
+    GETH_PORT=8555
+else
+    GETH_PORT=8545
+fi
+
+PARITY_VERSION=v1.8.0
+GETH_VERSION=v1.7.3-instantseal
+
+client_running() {
+    nc -z localhost "$GETH_PORT"
+}
+
+start_testrpc() {
+    if [ "$SOLIDITY_COVERAGE" = true ]; then
+    node_modules/.bin/testrpc-sc -i 16 --gasLimit 0xfffffffffff --port "$GETH_PORT"  > /dev/null &
+    else
+    node_modules/.bin/ganache-cli -i 15 --gasLimit 50000000 > /dev/null &
+    fi
+
+    rpc_pid=$!
+}
+
+check_docker() {
+    # check that docker exists before attempting to pull/run images
+    if ! [ -x "$(command -v docker)" ]; then
+        echo 'Error: docker is not installed' >&2
+        exit 1
+    fi
+}
+
+start_parity() {
+    check_docker
+    # pull the most stable release of parity
+    docker pull purta/parity-instantseal:$PARITY_VERSION
+    # run the container in detached mode
+    docker run -d -p 8545:8545 purta/parity-instantseal:$PARITY_VERSION
+
+    sleep 5
+}
+
+start_geth() {
+    check_docker
+    # pull the latest image using the dev test network
+    docker pull purta/geth-devnet:$GETH_VERSION
+    # run the geth dev network container
+    docker run -d -p 8545:8545 purta/geth-devnet:$GETH_VERSION
```

Comment:
Why did you have to create your own images? I think we should eventually move them under aragon's docker account

Reply:
I had to make my own images so that all the accounts needed for testing could be set up correctly. For geth I ended up having to have a way to build images for a PoA network and the instant seal dev network with accounts being set up on the fly. 

I agree that they should be moved eventually. The code for the geth images are [here](https://github.com/cpurta/geth-devnet) and the parity images are [here](https://github.com/cpurta/parity-instantseal).

Reply:
Perfect. We should create a separate issue for this, as IMO is not a blocker at this point

Reply:
Just published it to docker.io/aragon/geth-devnet

Reply:
Happy to use the new image using the aragon namespace, however I see that there is only a `latest` tag on docker hub. Do you want to use that tag or versioned tag like the one that is already being used (`v1.7.3`)?